client/login/wp-login/footer.jsx

```diff
@@ -0,0 +1,70 @@
+/**
+ * External dependencies
+ */
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames';
+import { connect } from 'react-redux';
+import { localize } from 'i18n-calypso';
+
+/**
+ * Internal dependencies
+ */
+import { getCurrentOAuth2Client } from 'state/ui/oauth2-clients/selectors';
+
+class LoginFooter extends Component {
```

Comment:
This looks good. Did you consider swapping `Component` for [`PureComponent`](https://facebook.github.io/react/docs/react-api.html#react.purecomponent) or even a [Functional Component](https://facebook.github.io/react/docs/react-api.html#react.purecomponent) like this:

```jsx
function LoginFooter( props ) {
  // ...
}
```

No need to make that change, just FYI.

Reply:
I didn't know about `PureComponent`, it's good to know!